x-pack/plugins/security_solution/public/common/components/exceptions/add_exception_comments.tsx

```diff
@@ -0,0 +1,128 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { memo, useState, useCallback, useMemo } from 'react';
+import styled, { css } from 'styled-components';
+import {
+  EuiTextArea,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiAvatar,
+  EuiAccordion,
+  EuiCommentList,
+  EuiCommentProps,
+  EuiText,
+} from '@elastic/eui';
+import { Comment } from '../../../lists_plugin_deps';
+import * as i18n from './translations';
+import { useCurrentUser } from '../../lib/kibana';
+import { getFormattedComments } from './helpers';
+
+interface AddExceptionCommentsProps {
+  exceptionItemComments?: Comment[];
+  newCommentValue: string;
+  newCommentOnChange: (value: string) => void;
+}
+
+const commentAccordionButtonClassname = 'exceptionCommentAccordionButton';
```

Comment:
super nit: For constants I think I've seen us use all uppercase usually like something like `COMMENT_ACCORDION_BUTTON_CLASS_NAME`